e2e/portal/pages/PaymentPage.ts

```diff
@@ -136,6 +140,27 @@ class PaymentPage extends BasePage {
 
     await this.importFileButton.click();
   }
+
+  async navigateToPaymentLog(): Promise<void> {
+    await this.paymentLogTab.click();
+    await this.page.waitForURL('**/payment-log');
+  }
+
+  async validatePaymentLogEntries(expectedNote: string): Promise<void> {
+    const rows = this.paymentLogTable.locator('tbody tr');
+    await expect(rows).toHaveCount(2);
+
+    const noteRow = rows.filter({ hasText: expectedNote });
+    const createdRow = rows.filter({ hasText: 'Created' });
+
+    await expect(noteRow).toBeVisible();
+    await expect(createdRow).toBeVisible();
+  }
+
+  async validatePaymentNote(expectedNote: string): Promise<void> {
```

Comment:
I think thoughtout the chain of these function and the name of the new test file it's not really clear that:
- the `validatePaymentNote` function also checks if a createdPayment event is there
- that the e2e test with file name `ViewPaymentNote.spec.ts` also checks the created Payment event

Maybe it makes sense to rename the file to ViewPaymentEvents.spec.ts and also than the related functions and test step description

Reply:
done (changed to ViewPaymentLog to stick more with the front-end naming of the subpage)